lib/signaling/v2/livetranscriptionsignaling.js

```diff
@@ -4,6 +4,7 @@ const MediaSignaling = require('./mediasignaling');
 
 /**
  * @emits LiveTranscriptionSignaling#transcription
+ * @property {boolean} isEnabled - Whether the connection is live and ready to receive transcription events
```

Comment:
Why do we need this?

Reply:
Since `isLiveTranscriptionEnabled` no longer exists, I removed it as well